Directory.Build.props

```diff
@@ -23,9 +23,6 @@
 		<ContinuousIntegrationBuild Condition="'$(GITHUB_ACTIONS)' == 'true'">true</ContinuousIntegrationBuild>
 		<EmbedUntrackedSources>true</EmbedUntrackedSources>
 	</PropertyGroup>
```

Comment:
Why do we remove the Deterministic builds?

Reply:
source link is included in the .NET SDK and `Microsoft.SourceLink.GitHub` is no longer needed. 

See <https://github.com/dotnet/sourcelink?tab=readme-ov-file#using-source-link-in-net-projects>:
> Starting with .NET 8, Source Link for the following source control providers is included in the .NET SDK and enabled by default:
> * [GitHub](http://github.com/) or [GitHub Enterprise](https://enterprise.github.com/home)
> * \[...]